src/domain/registration/form/RegistrationForm.tsx

```diff
@@ -33,14 +30,49 @@ import { profileQuery as ProfileQueryType } from '../../api/generatedTypes/profi
 import LoadingSpinner from '../../../common/components/spinner/LoadingSpinner';
 import NavigationConfirm from '../../../common/components/confirm/NavigationConfirm';
 import { GuardianInput, Language } from '../../api/generatedTypes/globalTypes';
-import { validateEmail } from '../../../common/components/form/validationUtils';
+import FormikDropdown, {
+  HdsOptionType,
+} from '../../../common/components/form/fields/dropdown/FormikDropdown';
+import { RegistrationFormValues } from '../types/RegistrationTypes';
+
+const schema = yup.object().shape({
+  guardian: yup.object().shape({
+    firstName: yup
+      .string()
+      .required('validation.general.required')
+      .max(255, 'validation.maxLength'),
+    lastName: yup
+      .string()
+      .required('validation.general.required')
+      .max(255, 'validation.maxLength'),
+    email: yup
+      .string()
+      .email('registration.form.guardian.email.input.error')
+      .required('validation.general.required'),
+    phoneNumber: yup
+      .string()
+      .required('validation.general.required')
+      .max(255, 'validation.maxLength'),
+    language: yup.string().max(255, 'validation.maxLength'),
+  }),
+  children: yup.array().of(
+    yup.object().shape({
+      postalCode: yup.string().required('validation.general.required'),
+      // relationship: yup.object().shape({
+      //   type: yup.string().required('validation.general.required').nullable(),
+      // }),
```

Comment:
Commented code that can likely be removed.

Reply:
Or fixed to actually work ..